### DISCUSSION TITLE
Termux /data/data/com.termux/home

### DISCUSSION THREAD

**Discussion by OW3N-JR** [termux/termux-app#3596] [Q&A] [answered]
So since I was running out of space I decided to delete Kali Linux from termux but before that I moved the Kali Linux tar file to Android so that I could install it another time.then I moved the file to /data/data/com.termux/home now I can't find it and every time I put in the command -$ cd /data/data/com.termux/home it says permission denied can someone please help me out with this.
👇 👇 here is the screenshot
![Screenshot_20230924-033806](https://github.com/termux/termux-app/assets/145691794/1ec7d033-52ff-4d4b-94a1-992609c12662)

**Comment by twaik** [comment]
We are not helping people with hacking tools.
https://wiki.termux.com/wiki/Hacking

**Comment by agnostic-apollo** [comment, marked as answer]
Termux `$HOME` directory is `/data/data/com.termux/files/home`. If you messed up ownership with messing with root or kali, you are on your own. You can check https://github.com/termux/termux-app/issues/2080#issuecomment-846575028 for potential fix or reinstall termux.

**Reply by sinuninu75-cloud** [reply]
👍
